k8s-operator/apis/v1alpha1/types_proxyclass.go

```diff
@@ -36,10 +36,24 @@ type ProxyClassList struct {
 }
 
 type ProxyClassSpec struct {
+	// Configuration for tailscaled running in the proxy.
+	// +optional
+	TailscaledConfig *TailscaledConfig `json:"tailscaledConfig,omitempty"`
 	// Proxy's StatefulSet spec.
-	StatefulSet *StatefulSet `json:"statefulSet"`
+	// +optional
+	StatefulSet *StatefulSet `json:"statefulSet,omitempty"`
 }
 
+type TailscaledConfig struct {
+	// AcceptRoutes can be set to true to make the proxy to accept routes.
+	// from subnet routers and route traffic via exit nodes (defaults to false).
+	// https://tailscale.com/kb/1019/subnets
```

Comment:
I don't think this actually impacts exit nodes, since they have to be explicitly chosen by the client?

```suggestion
	// AcceptRoutes can be set to true to make the proxy pod use
	// routes announced by other Tailscale nodes (e.g. subnet routers,
	// app connectors).
	// https://tailscale.com/kb/1019/subnets
```